app/models/event_log.rb

```diff
@@ -64,7 +68,14 @@ def entry
     EVENTS.detect { |event| event.id == event_id }
   end
 
+  def ip_address_string
+    inet_ntoa self.ip_address
+  end
+
   def self.record_event(user, event, options = {})
+    if options[:ip_address]
+      options[:ip_address] = inet_aton options[:ip_address]
```

Comment:
We prefer to call functions in this way with parentheses.
```ruby
options[:ip_address] = inet_aton(options[:ip_address])
```

We should probably add this to the styleguide and Rubocop.